packages/app-project/src/screens/ClassifyPage/ClassifyPage.js

```diff
@@ -89,7 +100,7 @@ function ClassifyPage({
           gap='medium'
           pad='medium'
         >
-          <Box as='main' fill='horizontal'>
+          <Box as='main' height={{ min: '400px'}} width='100%'>
```

Comment:
Gave the classifier area a min-height so the layout jump is not so jarring once lib-classifier is actually imported and displayed. The Classifier loading UX will eventually be enhanced. See https://github.com/zooniverse/front-end-monorepo/issues/6484